taichi/backends/cuda/codegen_cuda.cpp

```diff
@@ -77,7 +83,7 @@ class CodeGenLLVMCUDA : public CodeGenLLVM {
       for (auto task : offloaded_local) {
         TI_TRACE("Launching kernel {}<<<{}, {}>>>", task.name, task.grid_dim,
                  task.block_dim);
-
+        // TODO: It seems safe to get the Context pointer from |ctx_builder|?
```

Comment:
Yes, it seems so to me.

When launching a CUDA kernel `Context` will be copied by value.

Let's stick to what we have now and upgrade after `program.context` is removed :-)